nixos/modules/system/boot/loader/generic-extlinux-compatible/default.nix

```
@@ -30,6 +30,15 @@ in
         '';
       };
 
+      useDeviceTree = mkOption {
+        default = true;
+        type = types.bool;
+        description = ''
+          Whether to generate Device Tree-related directives in the
+          extlinux configuration.
+        '';
+      };
```

```suggestion
      useGenerationDeviceTree = mkOption {
        default = true;
        type = types.bool;
        description = ''
          Whether to generate Device Tree-related directives in the
          extlinux configuration.
          
          When enabled, the device tree binaries from the generation's kernel
          will be attempted to be loaded by the bootloader.
          
          Note that this affects all generations, regardless of the setting value
          at that specific point in time.
        '';
      };
```

Not entire sure about the phrasing, but it is important to explain that this also affects *previous generations* when generating the boot entries.

Any better idea for phrasing?

---

Good idea. For the last sentence, maybe we can have something like "Note that this affects all generations, regardless of the setting value *used in their configurations*?"

---

Sounds good.

---

Ok, reworded a little bit and pushed the new commits.